dist/goaccess/obs-goaccess-last-month.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+# Generates goaccess HTML data from last months rotated apache access logs
+# Runs from cron.d on every first of the month
```

Comment:
This comment refers to when this script is run. Shouldn't it be placed into the `dist/goaccess/obs-goaccess-last-month.crontab` file?

Also, in that `.contrab` file, this line `0 0 5 * *` is not exactly the same as what the comment says: ` # Runs from cron.d on every first of the month`...